### DISCUSSION TITLE
How do you format the date in pnp search results please?

### DISCUSSION THREAD

**Discussion by steveh2023** [microsoft-search/pnp-modern-search#3466] [Q&A]
Hi,

How do you format the date in pnp search results please?

In search results the date and time is showing - i only want to show the date ie not the time - how to do this please?

Thanks.

**Comment by kasperbolarsen** [comment]
Hi Steve,
The answer is .....is depends :-)
If you are using HandleBars this might be useful: https://docs.celigo.com/hc/en-us/articles/360045564992-Handlebar-expressions-for-date-and-time-format-codes

If you are using Adaptive Cards, this is my go to URL: https://stackoverflow.com/questions/64020012/adaptive-card-date-format-dd-mm-yyyy

**Reply by steveh2023** [reply]
Hi,

Thank you Kasperbolarsen for the info - The 1st link is really useful.

This expression: {{{dateFormat "ll" timeStamp}}} is the one that i want and this: {{0DE-Merlin-PublicationDate}} is the date field...

Unfortunately I do not know how to apply one to the other, I tried this:

{{0DE-Merlin-PublicationDate}}{{{dateFormat "ll" timeStamp}}} and i get: "Error: Missing helper: "dateFormat""

IF you or anyone knows the correct syntax that would be handy - thanks!

**Comment by wobba** [comment]
Answered in #3468
